benchmarks/DL3/benchmarks_DL3_IRFs_and_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,923 @@
+    "As with any other part of _protopipe_ and being part of the official repository, this notebook can be further developed by any interested contributor.  \n",
+    "The execution of this notebook is not currently automatic, it must be done locally by the user - preferably _before_ pushing a pull-request.  \n",
+    "**IMPORTANT:** Please, if you wish to contribute to this notebook, before pushing anything to your branch (better even before opening the PR) clear all the output and remove any local directory paths that you used for testing (leave empty strings).\n",
+    "\n",
+    "**TODO:**  \n",
+    "* update everything..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of contents\n",
+    "\n",
+    "* [Optimized cuts](#Optimized-cuts)\n",
+    "    - [Direction cut](#Direction-cut)\n",
+    "* [Differential sensitivity from cuts optimization](#Differential-sensitivity-from-cuts-optimization)\n",
+    "* [IRFs](#IRFs)\n",
+    "    - [Effective area](#Effective-area)\n",
+    "    - [Point Spread Function](#Point-Spread-Function)\n",
+    "        + [Angular resolution](#Angular-resolution)\n",
+    "    - [Energy dispersion](#Energy-dispersion)\n",
+    "        + [Energy resolution](#Energy-resolution)\n",
+    "    - [Background rate](#Background-rate)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import pyirf\n",
+    "import numpy as np\n",
+    "from astropy.io import fits\n",
+    "import uproot4 as uproot\n",
+    "import astropy.units as u\n",
+    "import matplotlib.pyplot as plt\n",
+    "from astropy.table import QTable, Table, Column\n",
+    "from matplotlib.ticker import ScalarFormatter\n",
+    "\n",
+    "from pyirf.binning import bin_center\n",
+    "from pyirf.utils import cone_solid_angle\n",
+    "\n",
+    "%matplotlib inline\n",
+    "plt.rcParams['figure.figsize'] = (9, 6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Input data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Protopipe performance"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Add Path to the performance folder\n",
+    "protopipe_file = './performance_protopipe_CTAnorth_full_array_Zd20_South_50h.fits.gz'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load performance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MARS performance (available here: https://forge.in2p3.fr/projects/step-by-step-reference-mars-analysis/wiki)"
+   ]
```

Comment:
could you add at the beginning also the Prod3b page URL?
There we have all ED IRFs

Reply:
I've already put both page URLs (for MARS and ED)
![Screenshot_2020-12-16 benchmarks_DL3_IRFs_and_sensitivity_pyirf - Jupyter Notebook](https://user-images.githubusercontent.com/51763341/102298523-90d96800-3f51-11eb-8893-7638d7571199.png)
)

Reply:
oopsie! Apologies, I didn't see it.
checking diffs on notebooks is less clear than plain code...
actually there should be something called [ReviewNB](https://www.reviewnb.com/) that was working before, but I don't see it here....